sdk/include/ndk/rtlfuncs.h

```diff
@@ -1568,6 +1568,15 @@ RtlReleasePrivilege(
     _In_ PVOID ReturnedState
 );
 
+NTSYSAPI
+NTSTATUS
+NTAPI
+RtlRemovePrivileges(
+    IN HANDLE TokenHandle,
+    IN PULONG PrivilegesToKeep,
+    IN ULONG PrivilegeCount
+);
+
```

Comment:
This RTL function exported from ntdll.dll in user-mode, I not found NTDDI_VERSION macro used like this in ntdll.dll, shall I do so?

Reply:
What's the point of using NTDDI_VERSION? It will just make things more complicated. If you want to use it from anywhere else, you now have to either redefine NTDDI_VERSION or duplicate the prototype.